engine/engine.go

```
@@ -208,6 +208,8 @@ func (e *Engine) processMessageMediaChunk(ctx context.Context, msg *sarama.Consu
 	if err := json.Unmarshal(msg.Value, &mediaChunk); err != nil {
 		return errors.Wrap(err, "unmarshal message value JSON")
 	}
+	inputMessage, _ := json.Marshal(mediaChunk)
```

Even though this will probably always work, we never ignore errors. :) Please handle this error.

---

Yeah, I updated. thanks!